cluster-autoscaler/simulator/predicates.go

```diff
@@ -125,6 +125,7 @@ func NewPredicateChecker(kubeClient kube_client.Interface, stop <-chan struct{})
 		NoOpEventRecorder{},
 		stop,
 		scheduler.WithAlgorithmSource(algorithmSource),
+		scheduler.WithFrameworkConfigProducerRegistry(nil),
```

Comment:
you can remove line 127 above, by default the scheduler uses "schedulerconfig.SchedulerDefaultProviderName"

Reply:
That's a fair point. I was wondering if we're better off not relying on default, but assuming that by default we'll get a DefaultProvider seems reasonable.